server/writer/writer.go

```diff
@@ -72,7 +72,13 @@ func JSON(w http.ResponseWriter, code int, v interface{}, pretty bool) {
 	}
 }
 
+// JSONOK is a helper for status "200 OK" responses
+func JSONOK(w http.ResponseWriter, v interface{}, pretty bool) {
+	JSON(w, http.StatusOK, v, pretty)
```

Comment:
ℹ️ Turns out there is 1 (!) caller of `writer.JSON` where status is not `http.StatusOK` -- the health check handler.